crates/wit-bindgen-wasmcloud-provider-binary/Cargo.lock

```diff
@@ -0,0 +1,396 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
```

Comment:
Why is there a `Cargo.lock`, shouldn't this be a part of the workspace?

Reply:
- [ ] Remove Cargo.lock